### PR TITLE
Bootstrap and main page update

### PR DESCRIPTION
**Pull request by MaciekWin3**
Reopen of https://github.com/dotnet-websharper/templates/pull/48

### CODE REVIEW

**Comment by granicz** [comment]
Just getting to this now, sorry for the delay. @MaciekWin3 Your changes are for websharper-web only, any reason not to apply them to the other project templates that use the same starter app?

**Comment by MaciekWin3** [comment]
I noticed that only these templates are using an outdated version (3.3.4) of Bootstrap. To enhance the experience for new users, it would be better to update to a newer version. I also made some minor changes to the Site.cs and Site.fs files, such as using the ternary operator in the C# template, as I believe it makes the code cleaner.

File: WebSharper.Templates/templates/ClientServer-CSharp/Main.html [hunk 1]
```diff
@@ -1,79 +1,98 @@
...
-                For an enhanced template that provides automatic GitHub deployment to Azure, fork from <a href="https://github.com/intellifactory/ClientServer.Azure">GitHub</a>, or
-                read more <a href="https://websharper.com/blog-entry/4368/deploying-websharper-apps-to-azure-via-github">here</a>.
-            </p>
-        </div>
+      <div class="container">
+        <p class="text-muted text-center">
+          For an enhanced template that provides automatic GitHub deployment to
+          Azure, fork from
+          <a href="https://github.com/intellifactory/ClientServer.Azure"
+            >GitHub</a
```
Comment (granicz) [inline WebSharper.Templates/templates/ClientServer-CSharp/Main.html:82]: This looks weird - probably an artifact of autoformat. Can you move `>` to the preceding line, and the same for the closing tag? I'd just put all on the same line.

File: WebSharper.Templates/templates/ClientServer-CSharp/Main.html [hunk 2]
```diff
@@ -1,79 +1,98 @@
...
+          >, or read more
+          <a
+            href="https://websharper.com/blog-entry/4368/deploying-websharper-apps-to-azure-via-github"
+            >here</a
```
Comment (granicz) [inline WebSharper.Templates/templates/ClientServer-CSharp/Main.html:86]: Same here as above.

File: WebSharper.Templates/templates/ClientServer-FSharp/Main.html [hunk 3]
```diff
@@ -1,74 +1,98 @@
...
-                For an enhanced template that provides automatic GitHub deployment to Azure, fork from <a href="https://github.com/intellifactory/ClientServer.Azure">GitHub</a>, or
-                read more <a href="https://websharper.com/blog-entry/4368/deploying-websharper-apps-to-azure-via-github">here</a>.
-            </p>
-        </div>
+      <div class="container">
+        <p class="text-muted text-center">
+          For an enhanced template that provides automatic GitHub deployment to
+          Azure, fork from
+          <a href="https://github.com/intellifactory/ClientServer.Azure"
+            >GitHub</a
```
Comment (granicz) [inline WebSharper.Templates/templates/ClientServer-FSharp/Main.html:82]: See comment at the same line for the C# client-server project.

File: WebSharper.Templates/templates/ClientServer-FSharp/Main.html [hunk 4]
```diff
@@ -1,74 +1,98 @@
...
+          >, or read more
+          <a
+            href="https://websharper.com/blog-entry/4368/deploying-websharper-apps-to-azure-via-github"
+            >here</a
```
Comment (granicz) [inline WebSharper.Templates/templates/ClientServer-FSharp/Main.html:86]: Here too.

File: WebSharper.Templates/templates/ClientServer-FSharp/Site.fs [hunk 5]
```diff
@@ -15,9 +15,13 @@ module Templating =
     // Compute a menubar where the menu item for the given endpoint is active
     let MenuBar (ctx: Context<EndPoint>) endpoint : Doc list =
         let ( => ) txt act =
-             li [if endpoint = act then yield attr.``class`` "active"] [
-                a [attr.href (ctx.Link act)] [text txt]
-             ]
+            let isActive = if endpoint = act then "nav-link active" else "nav-link"
+            li [ attr.``class`` "nav-item" ] [
```
Comment (granicz) [inline WebSharper.Templates/templates/ClientServer-FSharp/Site.fs:19]: Please remove the spaces around the content in [].

File: WebSharper.Templates/templates/ClientServer-CSharp/Main.html (hunk 1 above)
Comment (MaciekWin3) [reply WebSharper.Templates/templates/ClientServer-CSharp/Main.html]: done

File: WebSharper.Templates/templates/ClientServer-CSharp/Main.html (hunk 2 above)
Comment (MaciekWin3) [reply WebSharper.Templates/templates/ClientServer-CSharp/Main.html]: done

File: WebSharper.Templates/templates/ClientServer-FSharp/Main.html (hunk 3 above)
Comment (MaciekWin3) [reply WebSharper.Templates/templates/ClientServer-FSharp/Main.html]: done

File: WebSharper.Templates/templates/ClientServer-FSharp/Main.html (hunk 4 above)
Comment (MaciekWin3) [reply WebSharper.Templates/templates/ClientServer-FSharp/Main.html]: done

File: WebSharper.Templates/templates/ClientServer-FSharp/Site.fs (hunk 5 above)
Comment (MaciekWin3) [reply WebSharper.Templates/templates/ClientServer-FSharp/Site.fs]: done

**Comment by granicz** [comment]
@MaciekWin3 Thanks!

**Pull request merged**